lib/ansible/plugins/inventory/netbox.py

```diff
@@ -303,7 +315,8 @@ def validate_query_parameters(self, x):
         v = tuple(x.values())[0]
 
         if not (k in ALLOWED_DEVICE_QUERY_PARAMETERS or k.startswith("cf_")):
-            self.display.warning("Warning: %s not in %s or starting with cf (Custom field)" % (k, ALLOWED_DEVICE_QUERY_PARAMETERS))
+            msg = "Warning: %s not in %s or starting with cf (Custom field)" % (k, ALLOWED_DEVICE_QUERY_PARAMETERS)
```

Comment:
`cf` could be quoted (if you are willing to perform this change, first `%s` could be replaced by `%r`)